### ISSUE TITLE
Suggest to add a second WAVEGUIDES.xml file for custom waveguide

### ISSUE THREAD

**Issue by JinWallner** [SiEPIC/SiEPIC-Tools#150]
Hi,

Can we add a second WAVEGUIDES.xml file, e.g. WAVEGUIDES_custom.xml for custom waveguide configuration? Currently, custom waveguides and PDK waveguides share the same file. It needs to be remerged for every new PDK release.

Thanks,

Jin

**Comment by mustafacc** [comment]
Good idea, perhaps instead of importing WAVEGUIDES.xml we should just import every xml file containing "WAVEGUIDE".

**Comment by lukasc-ubc** [comment]
I have implemented this.  I loads WAVEGUIDES.xml and then all WAVEGUIDE_*.xml files.  It is on GitHub, but not released in Package Manager. Can you please check?

**Comment by mustafacc** [comment]
Works okay for EBeam but there seems to be issues with AMF and AIM

AMF: generated waveguides missing pinrec layers + asterisk
![image](https://user-images.githubusercontent.com/8067688/124862535-69d49900-df6a-11eb-976e-3e90d88cc360.png)

AIM: waveguides don't generate properly
![image](https://user-images.githubusercontent.com/8067688/124862566-7fe25980-df6a-11eb-9969-fdc5853193d2.png)

**Comment by lukasc-ubc** [comment]
Hi Mustafa,
Thanks for checking.  I don’t see this problem on my end.  Let me investigate further.

**Comment by lukasc-ubc** [comment]
I found an issue where a string representation of the bezier parameter caused the waveguide to not generate. Just fixed. Before updating, can you check if this problem only occurred for Bezier curve waveguides?  i.e., you shouldn't have had this issue with non-bezier waveguides (e.g., TM).
